src/v2/ion/RemediationConstants.js

```diff
@@ -41,6 +41,7 @@ const FORMS = {
   DEVICE_APPLE_SSO_EXTENSION: 'device-apple-sso-extension',
   CANCEL_TRANSACTION: 'cancel-transaction',
   LAUNCH_AUTHENTICATOR: 'launch-authenticator',
+  DEVICE_ENROLLMENT: 'device-enrollment',
```

Comment:
let's change this to be more specific.. `DEVICE_ENROLLMENT_TERMINAL_VIEW`

Reply:
this meant for form name hence shall be same to form name unless we change form name in api.

Reply:
Miss that this is the Form we created only for UI. Make sense to change to `DEVICE_ENROLLMENT_TERMINAL_VIEW`